src/App.js

```diff
@@ -21,45 +21,159 @@ const generateSquares = () => {
       currentId += 1;
     }
   }
-
   return squares;
 };
 
 const App = () => {
-  // This starts state off as a 2D array of JS objects with
-  // empty value and unique ids.
   const [squares, setSquares] = useState(generateSquares());
+  console.log(`${squares[0][1].value}`);
```

Comment:
Remember to remove unneeded print statements when you're finished with a project.